playground/src/pages/CodingEnvironment.tsx

```diff
@@ -176,16 +68,19 @@ end`;
 
   const runProgram = async () => {
     init().then(() => {
+      const inputCheck = checkInputs(inputs);
+        if (!inputCheck.isValid) {
+          setOutput(inputCheck.errorMessage);
+          toast.error("Running failed");
```

Comment:
tiny nit: I would probably say "Execution failed"

Reply:
that is better than "Running" indeed